bundles/io/org.eclipse.smarthome.io.net/OSGI-INF/metatype/org.eclipse.smarthome.io.net.http.internal.SecureHttpClientFactory.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.osgi.org/xmlns/metatype/v1.1.0 http://www.osgi.org/xmlns/metatype/v1.1.0">
+
+	<OCD name="HTTP Client Factory" id="HttpClientFactory" description="HTTP Client Factory for generting jetty http clients">
+		<AD name="minThreads" id="minThreads" 
+		    default="10" min="1"
```

Comment:
Aren't 10 threads quite a lot (as a minimum) on a resource-constrained gateway? 
And shouldn't it even be `min="0"` here?

Reply:
This is a common thread pool for all jetty clients. Each jetty client needs at least four threads. So 10 is not really a lot.
And the threads are only started, if actually jetty client are created.